Nodejs/Product/Nodejs/Options/NodejsNpmOptionsPage.cs

```diff
@@ -47,6 +48,8 @@ public NodejsNpmOptionsPage()
         /// </summary>
         public bool ShowOutputWindowWhenExecutingNpm { get; set; }
 
+        public bool EnableAutomaticTypingsAcquisition { get; set; }
```

Comment:
Yes, I was referring to removing the UI option, and auto dts would be enabled by default for all NTVS projects (and you wouldn't be able to turn it off). The other benefit of this is that users will come to us and describe the issue they're running into so we can fix the problem that makes them want to turn it off. We can always add the option back then and put out a dev build if it is indeed problematic. 

But just to be super clear, the options are whatever we want them to be :wink: